test/cluster/rollout.sh

```diff
@@ -25,7 +25,13 @@ os::cmd::expect_success "oc project logging"
 os::log::info "Checking for DeploymentConfigurations..."
 for deploymentconfig in ${OAL_EXPECTED_DEPLOYMENTCONFIGS}; do
 	os::cmd::expect_success "oc get deploymentconfig ${deploymentconfig}"
-	os::cmd::expect_success "oc rollout status deploymentconfig/${deploymentconfig}"
+
+# this is to get around the current kubelet flake where we need to re-rollout to get the dc running
+#  for sanity sake and to not get stuck in the case of real issues, we will only do this once per dc
+	if [[ ! $(oc rollout status deploymentconfig/${deploymentconfig}) ]]; then
```

Comment:
Since you aren't using the stdout output of the command, you don't need the `$()` - you can just do
```
if ! oc rollout status deploymentconfig/${deploymentconfig} ; then
```